components/txn_types/src/lock.rs

```diff
@@ -85,6 +86,12 @@ pub struct Lock {
     // while committing is relatively expensive. So the solution is putting the ts of the rollback
     // to the lock.
     pub rollback_ts: Vec<TimeStamp>,
+
+    /// The commit TS of the latest PUT/DELETE record
+    pub last_change_ts: TimeStamp,
+    /// The number of versions that need skipping from the latest version to
```

Comment:
```suggestion
    /// The number of versions that need skipping from the current version to
```

Reply:
Due to similar reason, I think this change is not necessary.